djmoney/contrib/django_rest_framework/fields.py

```diff
@@ -40,6 +47,12 @@ def validate(self, value):
             amount = value.amount if isinstance(value, Money) else value
             return super(MoneyField, self).validate(amount)
 
+        def field_from_native(self, data, files, field_name, into):
+            super(MoneyField, self).field_from_native(data, files, field_name, into)
+            currency = data.get('{}_currency'.format(field_name), None)
```

Comment:
I think that sounds good. Django 1.11 will perhaps be important to support for quite some time because it will be the last "gateway" release for people to move from py2 to py3.

Anyways, if someone is dependent on old Django + old Python, then they might as well also use "old djmoney" :)